images/fcos-bastion-image/root/usr/bin/clean-registries.sh

```diff
@@ -1,11 +1,25 @@
 #!/bin/sh
 set -x
 
+wait_for_container_running() {
+  local container_name=$1
+  echo "Waiting for container $container_name to become running..."
+  while ! podman inspect -f '{{.State.Running}}' "$container_name" >/dev/null 2>&1; do
```

Comment:
```suggestion
  while ! [ $(podman inspect -f '{{.State.Health.Status}}' "$container_name" >/dev/null 2>&1) == "healthy" ]; do
```

Reply:
State.Health.Status not working

Reply:
:+1: let's find a way to check if the container is healthy, not just running.

Reply:
oh, it's probably because of the redirection